projects/route-analyzer/src/lib/app-route.ts

```diff
@@ -0,0 +1,13 @@
+/*!
+ * Copyright 2020 VMware, Inc.
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+export interface AppRoute {
```

Comment:
Please make sure we always add some documentation to public/exported objects/properties.

For example, what does component mean? what does canActivate mean?

Reply:
That's a great catch. I forgot the comments :)